<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Bridge : you &amp; me  | ToolsBugFlow</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.57.2" />
    
    
      <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
    

    
    
      <link href="/dist/css/app.d98f2eb6bcd1eaedb7edf166bd16af26.css" rel="stylesheet">
    

    

    
      
    

    

    <meta property="og:title" content="ToolsBugFlow" />
<meta property="og:description" content="Anaconda 1. CondaHTTPError SSLError Collecting package metadata (current_repodata.json): failed
CondaHTTPError: HTTP 000 CONNECTION FAILED for url https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/win-64/current_repodata.json Elapsed: -
An HTTP error occurred when trying to retrieve this URL. HTTP errors are often intermittent, and a simple retry will get you on your way. SSLError(MaxRetryError(&lsquo;HTTPSConnectionPool(host=\&lsquo;mirrors.tuna.tsinghua.edu.cn\&lsquo;, port=443): Max retries exceeded with url: /anaconda/cloud/conda-forge/win-64/current_repodata.json (Caused by SSLError(&ldquo;Can\&rsquo;t connect to HTTPS URL because the SSL module is not available.&ldquo;))&lsquo;))
PS C:\WINDOWS\system32&gt; conda create -n rl-tf python=3." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://horizonnn.github.io/post/toolsbugflow/" />
<meta property="article:published_time" content="2020-09-27T13:39:50+08:00" />
<meta property="article:modified_time" content="2020-09-27T13:39:50+08:00" />
<meta itemprop="name" content="ToolsBugFlow">
<meta itemprop="description" content="Anaconda 1. CondaHTTPError SSLError Collecting package metadata (current_repodata.json): failed
CondaHTTPError: HTTP 000 CONNECTION FAILED for url https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/win-64/current_repodata.json Elapsed: -
An HTTP error occurred when trying to retrieve this URL. HTTP errors are often intermittent, and a simple retry will get you on your way. SSLError(MaxRetryError(&lsquo;HTTPSConnectionPool(host=\&lsquo;mirrors.tuna.tsinghua.edu.cn\&lsquo;, port=443): Max retries exceeded with url: /anaconda/cloud/conda-forge/win-64/current_repodata.json (Caused by SSLError(&ldquo;Can\&rsquo;t connect to HTTPS URL because the SSL module is not available.&ldquo;))&lsquo;))
PS C:\WINDOWS\system32&gt; conda create -n rl-tf python=3.">


<meta itemprop="datePublished" content="2020-09-27T13:39:50&#43;08:00" />
<meta itemprop="dateModified" content="2020-09-27T13:39:50&#43;08:00" />
<meta itemprop="wordCount" content="130">



<meta itemprop="keywords" content="" />
<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="ToolsBugFlow"/>
<meta name="twitter:description" content="Anaconda 1. CondaHTTPError SSLError Collecting package metadata (current_repodata.json): failed
CondaHTTPError: HTTP 000 CONNECTION FAILED for url https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/win-64/current_repodata.json Elapsed: -
An HTTP error occurred when trying to retrieve this URL. HTTP errors are often intermittent, and a simple retry will get you on your way. SSLError(MaxRetryError(&lsquo;HTTPSConnectionPool(host=\&lsquo;mirrors.tuna.tsinghua.edu.cn\&lsquo;, port=443): Max retries exceeded with url: /anaconda/cloud/conda-forge/win-64/current_repodata.json (Caused by SSLError(&ldquo;Can\&rsquo;t connect to HTTPS URL because the SSL module is not available.&ldquo;))&lsquo;))
PS C:\WINDOWS\system32&gt; conda create -n rl-tf python=3."/>

  </head>

  <body class="ma0 avenir bg-near-white">

    
   
  

  <header>
    <div class="bg-black">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="https://horizonnn.github.io/" class="f3 fw2 hover-white no-underline white-90 dib">
      Bridge : you &amp; me
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/post/" title="Posts page">
              Posts
            </a>
          </li>
          
        </ul>
      
      











    </div>
  </div>
</nav>

    </div>
  </header>



    <main class="pb7" role="main">
      
  
  <article class="flex-l flex-wrap justify-between mw8 center ph3">

    <header class="mt4 w-100">
      <p class="f6 b helvetica tracked">
          
        POSTS
      </p>
      <h1 class="f1 athelas mb1">ToolsBugFlow</h1>
      
      <time class="f6 mv4 dib tracked" datetime="2020-09-27T13:39:50&#43;08:00">September 27, 2020</time>
      
      
    </header>

    <section class="nested-copy-line-height lh-copy serif f4 nested-links nested-img mid-gray pr4-l w-two-thirds-l">

<h1 id="anaconda">Anaconda</h1>

<h3 id="1-condahttperror-sslerror">1. CondaHTTPError  SSLError</h3>

<p>Collecting package metadata (current_repodata.json): failed</p>

<p>CondaHTTPError: HTTP 000 CONNECTION FAILED for url <a href="https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/win-64/current_repodata.json">https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/win-64/current_repodata.json</a>
Elapsed: -</p>

<p>An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
SSLError(MaxRetryError(&lsquo;HTTPSConnectionPool(host=\&lsquo;mirrors.tuna.tsinghua.edu.cn\&lsquo;, port=443): Max retries exceeded with url: /anaconda/cloud/conda-forge/win-64/current_repodata.json (Caused by SSLError(&ldquo;Can\&rsquo;t connect to HTTPS URL because the SSL module is not available.&ldquo;))&lsquo;))</p>

<p>PS C:\WINDOWS\system32&gt; conda create -n rl-tf python=3.6
Collecting package metadata (current_repodata.json): done
Solving environment: done</p>

<ul>
<li>解决办法：安装openssl并将其添加到环境变量中。</li>
</ul>

<h3 id="2-unsatisfiableerror">2. UnsatisfiableError</h3>

<ul>
<li><p>错误：</p>

<pre><code>UnsatisfiableError: The following specifications were found to be incompatible with each other:
Output in format: Requested package -&gt; Available versionsThe following specifications were found to be incompatible with your CUDA driver:
feature:/win-64::__cuda==11.0=0
feature:|@/win-64::__cuda==11.0=0
Your installed CUDA driver is: 11.0
</code></pre></li>

<li><p>原因：可能是conda源的问题</p></li>

<li><p>解决办法：采用<code>conda install -c conda-forge &lt;package_name&gt;</code>的安装方式</p></li>
</ul>
<ul class="pa0">
  
</ul>
<div class="mt6">
      
      
      </div>
    </section>

    <aside class="w-30-l mt6-l">




</aside>

  </article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://horizonnn.github.io/" >
    &copy; 2020 Bridge : you &amp; me
  </a>
    <div>










</div>
  </div>
</footer>

    

  <script src="/dist/js/app.3fc0f988d21662902933.js"></script>


  </body>
</html>
